apps/condo/domains/news/schema/GetNewsItemsRecipientsCountersService.test.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Generated by `createservice news.GetNewsItemsRecipientsCountersService --type mutations`
+ */
+const { map, pick } = require('lodash')
+
+const {
+    makeLoggedInAdminClient,
+    makeClient,
+    expectToThrowAuthenticationError, expectToThrowAccessDeniedErrorToResult,
+} = require('@open-condo/keystone/test.utils')
+
+const { getNewsItemsRecipientsCountersByTestClient } = require('@condo/domains/news/utils/testSchema')
+const {
+    createTestOrganization,
+    createTestOrganizationEmployeeRole,
+} = require('@condo/domains/organization/utils/testSchema')
+const { createTestOrganizationEmployee } = require('@condo/domains/organization/utils/testSchema')
+const { FLAT_UNIT_TYPE } = require('@condo/domains/property/constants/common')
+const { createTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { createTestResident } = require('@condo/domains/resident/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, createTestUser } = require('@condo/domains/user/utils/testSchema')
+
+const propertyMap = {
+    dv: 1,
+    type: 'building',
+    sections: [
+        {
+            id: '1',
+            type: 'section',
+            index: 1,
+            name: '1',
+            preview: null,
+            /*
+                 ___________________
+                | 33 | 34 | 35 | 36 |
+                |____|____|____|____|
+                | 29 | 30 | 31 | 32 |
+                |____|____|____|____|
+                | 25 | 26 | 27 | 28 |
+                |____|____|____|____|
+                | 21 | 22 | 23 | 24 |
+                |____|____|____|____|
+                | 17 | 18 | 19 | 20 |
+                |____|____|____|____|
+                | 13 | 14 | 15 | 16 |
+                |____|____|____|____|
+                |  9 | 10 | 11 | 12 |
+                |____|____|____|____|
+                |  5 |  6 |  7 |  8 |
+                |____|____|____|____|
+                |  1 |  2 |  3 |  4 |
+                |____|____|____|____|
+             */
```

Comment:
it's really needed here?)

Reply:
Why not? It's easy to understand building structure by visual data instead of trying to read code :)